client/components/Auth/InputField.vue

```diff
@@ -24,6 +30,7 @@
         </div>
         <select
           v-if="fieldType == 'dropdown'"
+          :id="fieldName"
```

Comment:
v-if on line 32 should be using `===`

Reply:
More code from main, I'll do a regex search & substitute to fix all occurances.